apps/example-orchestrator-ui/pages/_app.tsx

```diff
@@ -1,24 +1,44 @@
 import { AppProps } from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useState } from 'react';
 import { EuiProvider } from '@elastic/eui';
 import {
     defaultOrchestratorTheme,
     OrchestratorPageTemplate,
 } from '@orchestrator-ui/orchestrator-ui-components';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+// TODO: Ricardo -> I had to add this, or the tables would be unstyled?
```

Comment:
Oh dat is wel een beetje jammer, maar het lijkt er op dat deze er inderdaad in moet:
https://eui.elastic.co/#/guidelines/getting-started#setting-up-your-application